src/engraving/dom/stafftype.h

```diff
@@ -394,7 +394,7 @@ class StaffType
     bool m_genKeysig = true;       // create key signature at beginning of system
 
     // Standard: configurable properties
-    NoteHeadScheme m_noteHeadScheme = NoteHeadScheme::HEAD_NORMAL;
+    NoteHeadScheme m_noteHeadScheme = NoteHeadScheme::HEAD_AUTO;
```

Comment:
This change in default will have to be handled when reading and writing staves in MSCX files. Don't worry about doing that yet, but I'm adding this as a reminder for later.

When it comes time to handle this, my suggestion is as follows:

- When reading old files (created in MS 4.3 and earlier), if it says `normal` in the file, read it as `auto` instead to make sure the general style setting gets used.
- When reading new files (created in MS 4.4 and later), read the actual value (`auto`, `normal`, or whatever it may be) and trust this value.